Resources/ArmTemplates/template.json

```diff
@@ -105,6 +105,21 @@
         },
         "workspaceResourceGroup": {
             "type": "string"
+        },
+        "CmsRedisCacheConnectionString": {
```

Comment:
Please make this 'securestring' so credentials are not leaked via the Azure Portal